app/CMakeLists.txt

```diff
@@ -366,12 +352,26 @@ if(CELERITAS_BUILD_DEMOS AND CELERITAS_USE_Geant4)
   set(_demo_geant_integration_libs Celeritas::accel)
 
   if(CELERITAS_USE_ROOT)
-    celeritas_reflex_generate_dictionary(HitClasses
-      demo-geant-integration/HitClasses.hh
-      SELECTION demo-geant-integration/selection.xml
+    include_directories(
```

Comment:
I'd like to avoid propagating the DIRECTORY property to the other parts of this file: I wrapped it in a `function` to limit its scope. Is doing so infeasible or just awkward?

Reply:
I *really* don't want `include_directories` propagating around. By "temporary" you mean "until we no longer support all currently available versions of ROOT"... and we're likely to want at least two to five years of backward compatibility right? So I don't think this is a "temporary" problem.

Reply:
Thanks!